packages/react/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -54,7 +84,11 @@ const dropdownStyles: ComponentSlotStylesInput<DropdownProps & DropdownState, Dr
         },
         ':active': transparentColorStyle,
```

Comment:
Here again?

Reply:
same explanation as in https://github.com/stardust-ui/react/pull/863/files/35b41aea2d575722a43ff00db73186bc5d2aa442#r255066767